### DISCUSSION TITLE
Redirect saved on every publish

### DISCUSSION THREAD

**Discussion by danimalx** [Infocaster/UrlTracker#191] [General]
Running Umbraco 11.3.0
urlTracker 11.0.1

Every time a page is published urlTracker logs it as a redirect even when the URL has not changed. I now have 35000 redirects. Example below is just 3 publishes

![image](https://github.com/user-attachments/assets/1b89838b-d083-49dd-9c62-0f028e1a0480)

**Comment by D-Inventor** [comment]
Hi @danimalx !! Thank you for reaching out!

I recognize this issue. This issue has been resolved in the URL Tracker 13 for Umbraco 13. I would recommend that you upgrade your setup to Umbraco 13, since Umbraco 11 is already end-of-life. It has been addressed in this issue here: #161 .
